bin/sensei.js

```diff
@@ -0,0 +1,138 @@
+#! /usr/bin/env node
+const spawn = require("child_process").spawn;
+const fork = require("child_process").fork;
+const path = require("path");
+const WebpackDevServer = require("webpack-dev-server");
+const config = require("../webpack.config");
+const webpack = require("webpack");
+
+function cli(args) {
+  let trainingMaterialFolder = args[3];
+  switch (args[2]) {
+    case "serve":
+      serve(trainingMaterialFolder);
+      break;
+    case "build":
+      build(trainingMaterialFolder);
+      break;
+    case "pdf":
+      pdf(trainingMaterialFolder);
+      break;
+    case "help":
+    case "-h":
+    default:
+      help();
+      break;
+  }
+}
+
+function serve(trainingMaterialFolder = ".") {
+  console.log("Start dev server");
+  const server = new WebpackDevServer(
+    webpack(config({ material: trainingMaterialFolder }))
+  );
+  server.listen(8080, "0.0.0.0", function (err) {
+    if (err) {
+      console.log(err);
+    } else {
+      console.log(
+        "Navigate to http://localhost:8080/slides.html or http://localhost:8080/labs.html"
+      );
+    }
+  });
+}
+
+function build(trainingMaterialFolder = ".") {
+  console.log("Build slides & labs");
+  return new Promise((resolve, reject) => {
+    webpack(config({ material: trainingMaterialFolder }), (err, stats) => {
+      if (err || stats.hasErrors()) {
+        console.error("HTML files generation failed!", err);
+        reject();
+      } else {
+        console.log("Files generated to dist folder");
+        resolve();
+      }
+    });
+  });
+}
+
+async function pdf(trainingMaterialFolder) {
+  console.log("Generate pdf slides & labs");
+  await build(trainingMaterialFolder);
+  let trainingName = trainingMaterialFolder
+    ? path.basename(trainingMaterialFolder)
+    : "";
```

Comment:
Maybe we can use the current folder name here? To avoid to have filenames like Zenika-Formation--Slides.pdf

Reply:
Absolutly right ! I fix it in a new commit